clojure-mode.el

```diff
@@ -205,6 +210,13 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
         "--"
         ["Align expression" clojure-align]
         "--"
+        ("Threading macro refactorings"
+         ["Thread an existing threading macro once more" clojure-thread]
+         ["Unwind a threading macro once" clojure-unwind]
+         ["Fully thread the form at point using ->" clojure-thread-first-all]
+         ["Fully thread the form at point using ->>" clojure-thread-last-all]
```

Comment:
These two items should be the first two in the menu. And I think they should say simply:

> Thread a form with ->
> Thread a form with ->>